big-o-notation/big-o.js

```diff
@@ -0,0 +1,44 @@
+/* exported uniqueQuadratic, uniqueLinear */
+
+function uniqueLinear(words) {
+  const seen = {};                  // 1 * 1 = O(1)
+  const unique = [];                // 1 * 1 = O(1)
+  for (
+    let i = 0;                      // 1 * 1 = O(1)
+    i < words.length;               // 2 * n = O(2n)
+    i++                             // 2 * n = O(2n)
+  ) {
+    const word = words[i];          // 3 * n = O(3n)
+    if (!seen[word]) {              // 2 * n = O(2n)
+      seen[word] = true;            // 2 * 1 = O(2)
+      unique[unique.length] = word; // 4 * n = O(4n)
+    }
```

Comment:
Since we are in a loop that iterates through the whole list, everything in here should come out to (at least) O(n)